### PR TITLE
Feature/jl/gs/db insertion

### PR DESCRIPTION
**Pull request by gursham2001**

### CODE REVIEW

File: src/main/java/com/sparta/om/App.java [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
 
 public class App 
 {
     public static void main(String[] args) {
         Connection postgresConnection = ConnectionManager.connectToDB();
-        UserDAO userDAO = new UserDAO(postgresConnection);
+        DBController userDAO = new DBController(postgresConnection);
         ArrayList<EmployeeDTO> employees = EmployeeDAO.PopulateArray("src/main/resources/EmployeeRecords.csv");
-        System.out.println(employees.get(0).toString());
+//        System.out.println(employees.get(0).toString());
```
Comment (gursham2001) [inline src/main/java/com/sparta/om/App.java]: method main should be empty

File: src/main/java/com/sparta/om/DB/controller/DBController.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            e.printStackTrace();
+        }
+    }
+
+    public void insertUsersToTable(String filename) {
+        ArrayList<EmployeeDTO> validatedEmployees = EmployeeDAO.PopulateArray(filename);
+        for (int i = 0; i < validatedEmployees.size(); i++) {
+            EmployeeDTO record = validatedEmployees.get(i);
+            System.out.println(record.returnSQLReady());
+            // change 10 to current count
```
Comment (gursham2001) [inline src/main/java/com/sparta/om/DB/controller/DBController.java]: comment not needed

File: src/main/java/com/sparta/om/DB/controller/DBController.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                preparedStatement.setString(6, record.getGender());
+                preparedStatement.setString(7, record.getEmail());
+                preparedStatement.setDate(8, Utilities.DateConverter(record.getDateOfBirth()));
+                preparedStatement.setDate(9, Utilities.DateConverter(record.getDateOfJoining()));
+                preparedStatement.setInt(10, record.getSalary());
+                preparedStatement.execute();
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+
```
Comment (gursham2001) [inline src/main/java/com/sparta/om/DB/controller/DBController.java]: whitespace not needed

File: src/main/java/com/sparta/om/dto/EmployeeDTO.java [hunk 4]
```diff
@@ -23,7 +23,7 @@ public class EmployeeDTO {
     private int salary;
 
     public EmployeeDTO(String[] csvInput) {
-        this.emplID = csvInput[0];
+        this.emplID = Integer.parseInt(csvInput[0]);
```
Comment (gursham2001) [inline src/main/java/com/sparta/om/dto/EmployeeDTO.java]: Change Interger.parseInt to parseInt for consistency

File: src/main/java/com/sparta/om/dto/util/Utilities.java [hunk 5]
```diff
@@ -1,13 +1,18 @@
...
+import java.time.format.DateTimeFormatter;
 
 public class Utilities {
 
-    public static String DateConverter(LocalDate date){
-        SimpleDateFormat formatter = new SimpleDateFormat("yyyy/MM/dd");
-        String strDate = formatter.format(date);
-        return strDate;
+    public static Date DateConverter(LocalDate date){
+//        System.out.println(date);
```
Comment (gursham2001) [inline src/main/java/com/sparta/om/dto/util/Utilities.java]: Line 11 - 14 not needed

**Comment by JamalLindo** [comment]
You did a good job fixing these comments :D

**Pull request merged**